SWIGGYDEMO_REACT/swiggy-react-app/src/components/itemDetailsAndCart/components/cartItems/components/cartItem/components/actionButton/ActionButtonContainer.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import ActionButton from "./ActionButton";
+
+function ActionButtonContainer ({count, onClickPlus, pk, onClickMinus}) {
+
+    function increaseOnPlus (pk) {
```

Comment:
this function is just calling the other one and not doing anything extra, so not really needed.

Reply:
Called function inside onClick